doc/api/process.markdown

```diff
@@ -871,6 +881,39 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to it's
+parent process using `process.send()`. They will be received as a
+['message'](child_process.html#child_process_event_message)
+event on the parent's ChildProcess object.
+
+If node was not spawned with an IPC channel, `process.send()` will be undefined.
+
+
+## process.disconnect()
+
+Close the IPC channel to the parent process, allowing this child to exit
+gracefully once there are no other connections keeping it alive.
+
+Identical to the parent process's
+[ChildProcess.disconnect()](child_process.html#child_process_child_disconnect),
+see that method for more information.
```

Comment:
nit: This is a comma splice. I would just leave off `see that method for more information` entirely as it is implied by the presence of the link. But if you did want to include that text, change the comma to a semicolon, or change it to a period and write it as two separate sentences.